content/W2015/W15-event-coop.md

```diff
@@ -0,0 +1,19 @@
+Title: Co-op Showcase Panel
+Date: 2015-01-09 15:00
+Category: Events
+Slug: co-op-panel
+Status: draft
+Author: Melissa Tedesco
+Summary: Co-op experience panel for new students.
```

Comment:
"Co-op experience panel" sounds a bit awkward. I would suggest "Upper year students showcase their co-op experience to help junior students in their job hunt this term."